driver-core/src/test/unit/com/mongodb/internal/operation/FindOperationUnitSpecification.groovy

```diff
@@ -174,7 +173,7 @@ class FindOperationUnitSpecification extends OperationUnitSpecification {
         }
 
         then:
-        testOperation(operation, [3, 2, 0], expectedCommand, async, commandResult)
```

Comment:
Why was this changed?

Reply:
Using the `allowDiskUse` option can only be used for versions 3.4 and higher.

Reply:
Right, but this is more than just a test of allowDiskUse, and the rest of it should be tested with 3.2.  Just add another test case for allowDiskUse that only runs on 3.4+

Reply:
I agree the where clause was originally complex but seems harder now its doubled.  Perhaps use some groovy magic will make it easier to read:

``` 
 async << [null, true] * 10
 limit << [100, -100, 100, 0, 100] * 4
 batchSize << [10, 10, -10, 10, 0] * 4
 allowDiskUse << [null, true] * 10
 version << [[3,2,0], [3, 4,0]] * 10
```